packages/flutter_tools/tool/tool_coverage.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
As a general comment, can this code be factored into functions instead of having a giant main()?